openapi/openapi.bzl

```diff
@@ -21,7 +21,7 @@ def _new_generator_command(ctx, gen_dir, rjars):
   gen_cmd += " -cp {cli_jar}:{jars} io.swagger.codegen.SwaggerCodegen generate -i {spec} -l {language} -o {output}".format(
       java = java_path,
       cli_jar = ctx.file.codegen_cli.path,
-      jars = ":".join([j.path for j in rjars]),
+      jars = ":".join([j.path for j in rjars.to_list()]),
```

Comment:
can you confirm if these changes will still worth with 0.26 or does this require users to upgrade to 0.17?

Reply:
I tested with 0.26.1 and was able to build fine.

Reply:
That works for me then!